client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/RedisCommandEvaluation.java

```diff
@@ -0,0 +1,16 @@
+package org.evomaster.client.java.controller.internal.db;
+
+import org.evomaster.client.java.instrumentation.RedisCommand;
+
+/**
+ * This class will link a given RedisCommand to the result of the distance calculation for that commmand.
+ */
+public class RedisCommandEvaluation {
+    public final RedisCommand redisCommand;
```

Comment:
those should be private, with public getters. usually, we have public fields only for DTOs